src/components/select/select.ts

```diff
@@ -465,6 +466,7 @@ export default class IgcSelectComponent extends EventEmitterMixin<
         part="base"
         style=${styleMap({ position: this.positionStrategy })}
         @click=${this.handleClick}
+        aria-hidden="true"
```

Comment:
Kinda impossible to add /* eslint-disable lit-a11y/click-events-have-key-events */ here without dropping the entire template which kind of defeats the purpose of the checks, so went with `aria-hidden` which silences such rules. 
Not sure if I prefer adding extra aria attr where it shouldn't be needed in theory or having to refactor around eslint comments so only such cases (delegated click for other elements) can be handled.
// CC @rkaraivanov

Reply:
At least for now, I think we can stick with the extra attributes.
Either `aria-hidden` or `role="presentation"` based on the context and DOM